Library/Formula/golo.rb

```diff
@@ -21,7 +20,7 @@ class Golo < Formula
 
   def install
     if build.head?
-      rake "special:bootstrap"
+      system "./gradlew", "installDist"
       libexec.install %w[target/appassembler/bin target/appassembler/lib]
     else
       libexec.install %w[bin doc lib]
```

Comment:
I think this is where the build is failling—looks like `doc` isn't provided anymore in the devel version.